README.md

```diff
@@ -165,6 +166,7 @@ executables and libraries for that target.
 | Commodore                       | VIC-20             | `mos-vic20-clang`                |
 | CP/M-65                         | -                  | `mos-cpm65-clang`                |
 | Dodo 6502 Game System           | -                  | `mos-dodo-clang`                 |
+| Commodore GEOS                  | 64, 128            | ˋmos-geos-cbm-clanˋ              |
```

Comment:
`cbm-clang`

Reply:
On the road, will fox it tomorrow! Thanks a lot to both of you for your time. I learned a lot!!!